contrib/babelfishpg_tsql/src/hooks.h

```diff
@@ -3,6 +3,7 @@
 #include "postgres.h"
 #include "catalog/catalog.h"
 #include "parser/analyze.h"
+#include "tcop/cmdtag.h"
```

Comment:
why do we need this header for? Ideally required headers should be added to .c files only

Reply:
I've added this because in this function PLTsqlProcessTransaction's parameters we have this QueryCompletion, for that we need this header.